src/Service/EntryManager.php

```diff
@@ -267,13 +275,27 @@ public function restore(User $user, Entry $entry): void
         $this->dispatcher->dispatch(new EntryRestoredEvent($entry, $user));
     }
 
-    public function pin(Entry $entry): Entry
+    /**
+     * @param User|null $actor this should only be null if it is a system call
```

Comment:
Explain in the doxygen style `@brief` section that the ping is both doing pin and unpin.

Reply:
done, though without the `@brief`